go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/opencontainers/runc v0.1.1 // indirect
 	github.com/ory/dockertest v3.3.4+incompatible
 	github.com/ory/ladon v1.0.1
+	github.com/pborman/uuid v1.2.0
```

Comment:
another uuid lib? Can we reuse the one from twinj?

Reply:
should we add a cleanup ticket? with this one we have 5(!) uuid libraries within gosoline

Reply:
yes, plz! can we use our own gosoline uuid wrapper? Get rid of all the other usages?

Reply:
This was a typo. I'll revert the changes in this file.

Reply:
We have 2 
```bash
grep uuid go.mod | grep -v indirect
```